react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -395,21 +416,38 @@ const Map: React.FC<MapProps> = ({
         );
         setReportedBoundingBoxAcc(union_of_reported_bboxes);
         const is3D = views?.viewports?.[0]?.show3D ?? false;
+        const axesLayer = st_layers.find((e) => {
+            return e["@@type"] === "AxesLayer";
+        });
+        const isAxesLayer = typeof axesLayer !== "undefined";
+        // target: camera will look at either center of axes if it exists or center of data ("union_of_reported_bboxes")
+        const target = boundingBoxeCenter(
+            isAxesLayer
+                ? (axesLayer!["bounds"] as BoundingBox)
```

Comment:
I think we can use axesLayer?.["bounds"] instead of (axesLayer!["bounds"] to get rid of non-null assertion